src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('.tree');
+const allListItem = [...list.querySelectorAll('li')];
+
+allListItem.map(oneListItem => {
+  const span = document.createElement('span');
+
+  oneListItem.prepend(span);
+  span.append(span.nextSibling);
+});
+
+list.addEventListener('click', even => {
```

Comment:
```suggestion
list.addEventListener('click', event=> {
```

or evnt if event is already declared in upper scope